pkg/ui/package.json

```diff
@@ -4,7 +4,22 @@
   "main": "index.tsx",
   "repository": "github.com/cockroachdb/cockroach",
   "license": "MIT",
+  "eslintConfig": {
+    "extends": "@cockroachlabs",
+    "ignorePatterns": ["protos.d.ts", "protos.js"],
```

Comment:
Does it mean that having this `ignorePatterns` section, we won't be able to completely remove "eslintConfig" section from `package.json`? If yes, would it be better to move eslint configuration to separate configuration file?

Reply:
My intention is to keep this configuration section. I was hoping to avoid managing another external config file. Why do you prefer config file approach?

Reply:
I don't think its critical, just separation of concerns and easier to find configs in `rc`(.eslintrc) files rather than in `package.json`.